src/assets/v1.3.0/sign_message_lib.json

```diff
@@ -12,6 +12,7 @@
     "100": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
     "137": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
     "246": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "1287": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```

Comment:
This is the latest version, the others should be updated in a separate PR

Reply:
The moonriver deployments are part of this PR too. The PR has been merged and you can see the latest changes on the main branch. We will also release this soon to npm.